src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -975,25 +975,6 @@ public class ShowHideConsoleMenuItemConverter : IValueConverter
         #endregion
     }
 
-    public class ShowHidePreviewBubblesConverter : IValueConverter
```

Comment:
@mjkkirschner This is on the  WPF UI. It is basically removing the old feature. Does it hold relation to the graph?